### PR TITLE
New role, new PR

### PR DESCRIPTION
**Pull request by harryharrold**
Fixes #

Need for more front-end skills

## Description

- It's a job ad....
-

### CODE REVIEW

File: src/data/blog/Hiring-web-dev-092019.md [hunk 1]
```diff
@@ -26,44 +26,59 @@ author_facebook: null
...
 
-##Working mainly in HTML/CSS and Javascript  
+We’re looking for someone who:
 
-You'll have a flair for implementing user interfaces using web technologies. You'll fully understand that what you are designing are interactions, on screens that vary in size, not fixed images on a fixed canvas and enjoy producing interfaces that respond to user needs. Your tools will be HTML, CSS and Javascript and you’ll be passionate about how they work together. If you’re familiar with React, that would be an advantage. You’ll know how to work in a team with several developers and you may already have led some projects or project elements and will be keen to do more team leading.
+- works mainly in HTML/CSS and Javascript
+- has a flair for implementing user interfaces using web technologies. 
+- understands they are developing interactions, on screens that vary in size, that must be accessible to all their users. 
+- is passionate about how  HTML/CSS and Javascript work together, and familiarity with React would be an advantage0
```
Comment (neonleann) [inline src/data/blog/Hiring-web-dev-092019.md:40]: You have a rogue 0 at the end of the sentence.

File: src/data/blog/Hiring-web-dev-092019.md [hunk 2]
```diff
@@ -26,44 +26,59 @@ author_facebook: null
...
-- Be able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be);
-- Value agility in software development (experience of working in an agile environment is an advantage but not essential)
-- Be passionate about accessibility
-- Be keen to work in an organisation that delivers both commercial and social good projects
-- Be inspired by user-centred development (even if you don’t have practical experience of this approach)
-- Be committed to taking part in team activities and individual learning
+- comfortable communicating in a remote team;
+- happy taking part in conversations about technical subjects;
+- able to demonstrate that you enjoy expanding your knowledge and thinking critically (this can be via a degree, but 1/3rd of our team members don’t have one, so doesn’t have to be);
+- value agility in software development (experience of working in an agile environment is an advantage but not essential)
```
Comment (neonleann) [inline src/data/blog/Hiring-web-dev-092019.md:48]: I'm no fan of semi-colons at the end of list items, but the first three have them and the last four don't.

File: src/data/blog/Hiring-web-dev-092019.md (hunk 1 above)
Comment (harryharrold) [reply src/data/blog/Hiring-web-dev-092019.md:40]: Yes, damnit! Thank you!

...I've got to remember how to edit my commit now...

**Review by neonleann — APPROVED** [review]
Lovely apart from my minor gripes 👏.

File: src/data/blog/Hiring-web-dev-092019.md (hunk 2 above)
Comment (harryharrold) [reply src/data/blog/Hiring-web-dev-092019.md:48]: I've made the bullet list punctuation consisten - it was all over the place. Thanks.

File: src/data/blog/Hiring-web-dev-092019.md [hunk 3]
```diff
@@ -26,44 +26,59 @@ author_facebook: null
...
   
-A great opportunity for growth and enjoyment. Neontribe prides itself on skills development, great training budgets, flexible working, part-time hours and supporting return to work and other initiatives to help you enjoy your work. You’ll be stepping into the shoes of a developer who taught herself to code while she was working in retail and banking and then spent 3 years at Neontribe before heading to London as a front end developer for a major fashion brand.
+A great opportunity for growth and enjoyment. Neontribe prides itself on skills development, great training budgets, flexible working, part-time hours and supporting return to work and other initiatives to help you enjoy your work. 
 
 ##Salary  
 
-We’d expect to pay in the region of £24,786.00 - £29,532.00 for this role.
+We’d expect to pay in the region of £35,000.00 - £42,000.00 for this role.
 
 To Apply
```
Comment (Nikomus) [inline src/data/blog/Hiring-web-dev-092019.md:65]: Should this be a heading?

Comment (harryharrold) [reply src/data/blog/Hiring-web-dev-092019.md]: It Is A hEaDiNg. It HaS aLwAyS bEeN a HeAdInG...

Comment (harryharrold) [reply src/data/blog/Hiring-web-dev-092019.md]: It's totally a Heading. It was always a headling. I don't know what you mean....

(Fixed.)

**Pull request merged**